mlflow/tracking/_tracking_service/client.py

```diff
@@ -109,12 +109,20 @@ def list_run_infos(
             experiment_id, run_view_type, max_results, order_by, page_token
         )
 
-    def list_experiments(self, view_type=None):
+    def list_experiments(self, view_type=None, max_results=None, page_token=None):
         """
-        :return: List of :py:class:`mlflow.entities.Experiment`
+        :param max_results: If passed, specifies the maximum number of experiments desired.
+                            If not passed, all experiments will be returned for the File and
+                            SQLAlchemy backends. For the REST backend, the server will determine
+                            an appropriate number of experiments to return.
+        :param page_token: Token specifying the next page of results. It should be obtained from
+                            a ``list_experiments`` call.
+        :return: PagedList of :py:class:`mlflow.entities.Experiment`
```

Comment:
Can we update the return type docstring for `search_runs()` as well? https://github.com/mlflow/mlflow/pull/3881/files#r651172964

Reply:
@dbczumar Created a follow-up PR [here](https://github.com/mlflow/mlflow/pull/4455)